.github/ISSUE_TEMPLATES/04_thanks.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "💝 Thank you"
+about: "@octokit/webhooks.js is awesome 🙌"
```

Comment:
The package name is `@octokit/webhooks-definitions`. But I think there are consumers of this repository beyond just npm, so maybe we should call refer to it as just "Octokit Webhooks"?

Same below in lines 9 & 11